### ISSUE TITLE
GDI upgrade 

### ISSUE THREAD

**Issue by bolenacz** [IMXNOOBX/cs2-external-esp#16]
Hi, cool project, you're doing great.You can change the overlay to something more modern directx or imgui, because GDI takes 45-60 fps, you can also add a simple anti-aliasing aimbot that works only through the mouse

**Comment by bolenacz** [comment]
If someone knows how to work with overlays or can make a simple aimbot, then start pulling

**Comment by IMXNOOBX** [comment]
> Hi, cool project, you're doing great.You can change the overlay to something more modern directx or imgui, because GDI takes 45-60 fps, you can also add a simple anti-aliasing aimbot that works only through the mouse

Yes its a great idea. I will implement it when I have some free time.
